en/middleware/uorb.md

```diff
@@ -141,10 +141,27 @@ Make sure not to mix `orb_advertise_multi` and `orb_advertise` for the same topi
 
 The full API is documented in [platforms/common/uORB/uORBManager.hpp](https://github.com/PX4/PX4-Autopilot/blob/main/platforms/common/uORB/uORBManager.hpp).
 
+## Message Versioning
 
```

Comment:
Just FYI, this is how I mark up new content. On release I'll know the version number and replace this. You should also add a release note in https://docs.px4.io/main/en/releases/main.html , ideally with a link to any associated PRs.

I'll review properly when you tell me this is ready and/or when this moves out of draft. At that time would be good to link the various proposals that lead to this in the description.

Glad you are doing this and not me :-)

```suggestion

<Badge type="tip" text="main (PX4 v1.16+)" />

```

Reply:
@hamishwillee thanks for the early review (: I've applied your suggestions, and with some of the related PRs getting merged I'm coming back to these docs.

There's essentially 2 things to document here:
1. The new versioned messages (already started here)
2. The ROS 2 translation node (to do)

The README in [this PR](https://github.com/PX4/PX4-Autopilot/pull/24113) documents point 2 well, I will now try to capture those points here. Once that's done I will open this PR for review